### PR TITLE
Add initial support for a hygon csv attester

### PR DESCRIPTION
**Pull request by BaoshunFang**

### CODE REVIEW

File: attestation-agent/attester/Cargo.toml [hunk 1]
```diff
@@ -18,15 +18,22 @@ serde_json.workspace = true
...
 [features]
 default = ["all-attesters"]
-all-attesters = ["tdx-attester", "sgx-attester", "az-snp-vtpm-attester", "snp-attester"]
+all-attesters = ["tdx-attester", "sgx-attester", "az-snp-vtpm-attester", "snp-attester", "csv-attester"]
 
 tdx-attester = ["tdx-attest-rs"]
 sgx-attester = ["occlum_dcap"]
 az-snp-vtpm-attester = ["az-snp-vtpm"]
 snp-attester = ["sev"]
+csv-attester = ["csv-rs"]
```
Comment (jialez0) [inline attestation-agent/attester/Cargo.toml:39]: We should also include `codicon`, `hyper`, `hyper-tls` and `tokio` as optional dependencies of `csv-attester` here.

**Review by jialez0 — COMMENTED** [review]
@BaoshunFang Can you add a signature to the commit message? The format is like this:

```
Signed-off-by: Jiale Zhang <zhangjiale@linux.alibaba.com>
```

File: attestation-agent/attester/Cargo.toml (hunk 1 above)
Comment (BaoshunFang) [reply attestation-agent/attester/Cargo.toml]: Done

**Review by jialez0 — APPROVED** [review]
LGTM.

File: attestation-agent/attester/src/csv/mod.rs [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+    use super::*;
+
+    #[ignore]
+    #[test]
+    fn test_csv_get_evidence() {
+        let attester = CsvAttester::default();
+        let report_data: Vec<u8> = vec![0; 64];
+
+        let evidence = attester.get_evidence(report_data);
+        assert!(evidence.is_ok());
```
Comment (jialez0) [inline attestation-agent/attester/src/csv/mod.rs:110]: This should be:
```rust
let evidence = attester.get_evidence(report_data).await;
assert!(evidence.is_ok()
```
And this is why the CI failed.

Comment (BaoshunFang) [reply attestation-agent/attester/src/csv/mod.rs:110]: Done

**Pull request merged**